server/controllers/middlewares/auth/protectedRoute.js

```diff
@@ -0,0 +1,14 @@
+const jwt = require('jsonwebtoken');
```

Comment:
You can make destructure here to get verify direct
```suggestion
const { verify } = require('jsonwebtoken');
```